src/Frontend.hs

```diff
@@ -172,15 +172,15 @@ type AulaMain =
 
 aulaMain :: ActionM m => ServerT AulaMain m
 aulaMain =
-       runHandler Page.viewRooms
+       forceChangeInitialPassword (runHandler Page.viewRooms)
```

Comment:
note that "profile page" and "settings page" are two different things.

but i still don't get it.  i guess i'll try what i have in mind and i'll see what you mean in 10 minutes (-: